<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1094</width>
    <height>883</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Breast cancer prediction</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon.jpg</normaloff>
    <normalon>icon.jpg</normalon>icon.jpg</iconset>
  </property>
  <property name="accessibleName">
   <string/>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QPushButton" name="ButtonLoadImage">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>140</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>1</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(150, 150, 255);
</string>
    </property>
    <property name="text">
     <string>Load scan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonPredict">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>670</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(150, 150, 255);</string>
    </property>
    <property name="text">
     <string>Predict</string>
    </property>
   </widget>
   <widget class="QLabel" name="LabelAnswer">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>726</y>
      <width>1000</width>
      <height>55</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_photo">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>200</y>
      <width>945</width>
      <height>441</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_back">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1061</width>
      <height>1151</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="LabelTitle">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>30</y>
      <width>521</width>
      <height>48</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>breast cancer predict</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>100</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>histopathology</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_choose_type">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>100</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose type</string>
    </property>
   </widget>
   <widget class="QLabel" name="LabelAnswer2">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>800</y>
      <width>1000</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>label_back</zorder>
   <zorder>ButtonLoadImage</zorder>
   <zorder>ButtonPredict</zorder>
   <zorder>LabelAnswer</zorder>
   <zorder>label_photo</zorder>
   <zorder>LabelTitle</zorder>
   <zorder>comboBox</zorder>
   <zorder>label_choose_type</zorder>
   <zorder>LabelAnswer2</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
